curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/timestamp-microservice.md

```diff
@@ -107,7 +107,7 @@ If the input date string is invalid, the api returns an object having the struct
       assert.equal(data.error.toLowerCase(), 'invalid date');
     },
     (xhr) => {
-      throw new Error(xhr.responseText);
+      assert(xhr.responseJSON.error.toLowerCase() === 'invalid date');
```

Comment:
```suggestion
      assert(xhr.responseJSON.error.toLowerCase() === 'invalid date', 'received: ' + xhr.responseJSON.error);
```
How about this?  It should tell the user something if it's not the expected response.

Reply:
Where is this text showing up? I don't see it anywhere.

Reply:
Sure, lets just go with it

Reply:
Sorry, @moT01, I'm not sure why I thought this would work.  If we want to report to the user we'd need to do something like 

```
try {
  assert(xhr.responseJSON.error.toLowerCase() === 'invalid date');
} catch {
  console.error(xhr)
}
```
but I'm not sure that's necessary, given that they'll have error logs in the console when the status code is not 200.

TL;DR: let's revert my suggestion.